### PR TITLE
CLOUD-1215 improve documentation

### PR DESCRIPTION
**Pull request by MarkWareham**
Some improvements that could be made without asking questions, from review of docs

### CODE REVIEW

**Comment by AlanRoth** [comment]
Make sure to include Glossary.adoc in the nav.adoc, otherwise it will be ignored

**Review by AlanRoth — APPROVED** [review]

File: docs/modules/ROOT/pages/Getting Started.adoc [hunk 1]
```diff
@@ -32,7 +32,7 @@ Slovakia, Slovenia, Spain, Sweden
 
 After confirming you agree with the Terms & Conditions of your Payara Cloud account, click Subscribe to create your account and bill your card for the first 30 days of your subscription.
 
-NOTE: If you don’t complete the subscription sign up process in a timely manner, your session will timeout. Click the “restart” button to resume the process. It will automatically set you up with a Basic subscription but you can upgrade it to Standard or Premium at any time.
+NOTE: If you don’t complete the subscription sign up process in a timely manner, your session will timeout. Click the `restart` button to resume the process. It will automatically set you up with a Basic subscription but you can upgrade it to Standard or Premium at any time.
```
Comment (pdudits) [inline docs/modules/ROOT/pages/Getting Started.adoc:35]: https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/

Macro for correct consistent styling should be `[restart]`. Now why "Suggest an edit is not available here?" :)

Comment (AlanRoth) [reply docs/modules/ROOT/pages/Getting Started.adoc:35]: If thats something you would like to do, I can enable experimental features

**Review by pdudits — APPROVED** [review]
We

**Pull request merged**